docs-ref-services/servicebus.md

```diff
@@ -2,47 +2,200 @@
+```python
+import os
+from azure.servicebus import ServiceBusClient
+
+connection_str = os.environ['SB_CONN_STR']
+
+sb_client = ServiceBusClient.from_connection_string(connection_str)
+```
+
+
+## Service Bus Queues
+Service Bus Queues are an alternative to Storage Queues that might be
 useful in scenarios where more advanced messaging features are needed
 (larger message sizes, message ordering, single-operation destructive
 reads, scheduled delivery) using push-style delivery (using long
 polling).
 
-The service can use Shared Access Signature authentication, or ACS
-authentication.
+The service can use Shared Access Signature authentication.
+
+### Create queue
+This creates a new queue within the Service Bus namespace. If a queue of the same name already exists an error will be raised. 
+```python
+sb_client.create_queue("MyQueue")
+```
+Optional parameters to configure the queue behaviour can also be specified
+```python
+sb_client.create_queue(
+    "MySessionQueue",
+    requires_session=True  # Create a sessionful queue
+    max_delivery_count=5  # Max delivery attempts per message
+)
+```
+
+### Get a queue client
+A QueueClient can be used to send and receive messages from the queue, along with other operations.
+```python
+queue_client = sb_client.get_queue("MyQueue")
+```
+
+### Sending messages
+The queue client can send one or more messages at a time:
+```python
+from azure.servicebus import Message
+
+message = Message("Hello World")
+queue_client.send(message)
+
+message_one = Message("First")
+message_two = Message("Second")
+queue_client.send([message_one, message_two])
+```
+Each call to QueueClient.send will create a new service connection. To reuse the same connection for multiple send calls, you can open a sender:
+```python
+message_one = Message("First")
+message_two = Message("Second")
+
+with queue_client.get_sender() as sender:
+    sender.send(message_one)
+    sender.send(message_two)
+```
+
+### Receiving messages
+Message can be received from a queue as a continuous iterator. The default mode for message receiving is [PeekLock](https://docs.microsoft.com/rest/api/servicebus/peek-lock-message-non-destructive-read), which requires each message to be explicitly completed in order that it be removed from the queue.
+```python
+messages = queue_client.get_receiver()
+for message in messages:
+    print(message)
+    message.complete()
+```
+The service connection will remain open for the entirety of the iterator.
+If you find yourself only partially iterating the message stream, you should run the receiver in a `with` statement to ensure the connection is closed:
+```python
+with queue_client.get_receiver() as messages:
+    for message in messages:
+        print(message)
+        message.complete()
+        break
+```
+
+## Service Bus Topics and Subscriptions
+
+Service Bus topics are an abstraction on top of Service Bus Queues that
+make pub/sub scenarios easy to implement.
```

Comment:
Can you add just a little more here that explains why it's easier? Can be as short as possible, but as long as necessary to help the SDK user know whether they should use topics.

Minor: "Easy" is a subjective term we typically try to avoid.